### ISSUE TITLE
Upgrade python dependency in docker image

### ISSUE THREAD

**Issue by andriisoldatenko** [elastic/elasticsearch#44880]
**Steps to reproduce**:

 1.  `docker pull docker.elastic.co/elasticsearch/elasticsearch-oss:7.2.0`
 2. clair-scanner --ip=192.168.1.199 docker.elastic.co/elasticsearch/elasticsearch-oss:7.2.0

```
2019/07/25 23:20:42 [INFO] ▶ Start clair-scanner
2019/07/25 23:20:52 [INFO] ▶ Server listening on port 9279
2019/07/25 23:20:52 [INFO] ▶ Analyzing 21ce7dd52e6869f92d3a200a18b1dcca939ca431550d8c40f3581e867c0df177
2019/07/25 23:20:52 [INFO] ▶ Analyzing b28806d925ca721d333a35f82d8d36ca4c469b59213ec587f2816f5ab31a5b2c
2019/07/25 23:20:52 [INFO] ▶ Analyzing 89e50d8812763eb50d30b908bb54d7d39a0878447bf974c5f664a29d18f81fae
2019/07/25 23:20:52 [INFO] ▶ Analyzing acb2cea8184b1b8e9e09b92e30c1bf9c1692894d8662ec6b235137a3704f75aa
2019/07/25 23:20:52 [INFO] ▶ Analyzing 9b0b27958d8c1b5327ee34b4e37c7a0835b414328df95fbe7f717bca3242f37d
2019/07/25 23:20:52 [INFO] ▶ Analyzing 963472052b4507f37665bf6119296f228d1505245b02c330a2a36ffcd613b8b7
2019/07/25 23:20:52 [INFO] ▶ Analyzing fa01de787c2f723dfc6be6af08aca2f74fb9fa96a1f13717f636c5cb431385c9
2019/07/25 23:20:53 [WARN] ▶ Image [docker.elastic.co/elasticsearch/elasticsearch-oss:7.2.0] contains 3 total vulnerabilities
2019/07/25 23:20:53 [ERRO] ▶ Image [docker.elastic.co/elasticsearch/elasticsearch-oss:7.2.0] contains 3 unapproved vulnerabilities
+------------+---------------------+--------------+-----------------+-------------------------------------------------+
| STATUS     | CVE SEVERITY        | PACKAGE NAME | PACKAGE VERSION | CVE DESCRIPTION                                 |
+------------+---------------------+--------------+-----------------+-------------------------------------------------+
| Unapproved | High RHSA-2019:1587 | python-libs  | 2.7.5-77.el7_6  | Python is an interpreted, interactive,          |
|            |                     |              |                 | object-oriented programming language, which     |
|            |                     |              |                 | includes modules, classes, exceptions, very     |
|            |                     |              |                 | high level dynamic data types and dynamic       |
|            |                     |              |                 | typing. Python supports interfaces to many      |
|            |                     |              |                 | system calls and libraries, as well as to       |
|            |                     |              |                 | various windowing systems. Security Fix(es):    |
|            |                     |              |                 | * python: regression of CVE-2019-9636 due       |
|            |                     |              |                 | to functional fix to allow port numbers in      |
|            |                     |              |                 | netloc (CVE-2019-10160) For more details        |
|            |                     |              |                 | about the security issue(s), including the      |
|            |                     |              |                 | impact, a CVSS score, acknowledgments, and      |
|            |                     |              |                 | other related information, refer to the CVE     |
|            |                     |              |                 | page(s) listed in the References section.       |
|            |                     |              |                 | https://access.redhat.com/errata/RHSA-2019:1587 |
+------------+---------------------+--------------+-----------------+-------------------------------------------------+
| Unapproved | High RHSA-2019:1587 | python       | 2.7.5-77.el7_6  | Python is an interpreted, interactive,          |
|            |                     |              |                 | object-oriented programming language, which     |
|            |                     |              |                 | includes modules, classes, exceptions, very     |
|            |                     |              |                 | high level dynamic data types and dynamic       |
|            |                     |              |                 | typing. Python supports interfaces to many      |
|            |                     |              |                 | system calls and libraries, as well as to       |
|            |                     |              |                 | various windowing systems. Security Fix(es):    |
|            |                     |              |                 | * python: regression of CVE-2019-9636 due       |
|            |                     |              |                 | to functional fix to allow port numbers in      |
|            |                     |              |                 | netloc (CVE-2019-10160) For more details        |
|            |                     |              |                 | about the security issue(s), including the      |
|            |                     |              |                 | impact, a CVSS score, acknowledgments, and      |
|            |                     |              |                 | other related information, refer to the CVE     |
|            |                     |              |                 | page(s) listed in the References section.       |
|            |                     |              |                 | https://access.redhat.com/errata/RHSA-2019:1587 |
+------------+---------------------+--------------+-----------------+-------------------------------------------------+
| Unapproved | High RHSA-2019:1619 | vim-minimal  | 2:7.4.160-5.el7 | Vim (Vi IMproved) is an updated and improved    |
|            |                     |              |                 | version of the vi editor. Security Fix(es):     |
|            |                     |              |                 | * vim/neovim: ':source!' command allows         |
|            |                     |              |                 | arbitrary command execution via modelines       |
|            |                     |              |                 | (CVE-2019-12735) For more details about the     |
|            |                     |              |                 | security issue(s), including the impact,        |
|            |                     |              |                 | a CVSS score, acknowledgments, and other        |
|            |                     |              |                 | related information, refer to the CVE           |
|            |                     |              |                 | page(s) listed in the References section.       |
|            |                     |              |                 | https://access.redhat.com/errata/RHSA-2019:1619 |
+------------+---------------------+--------------+-----------------+-------------------------------------------------+
```

**Comment by andriisoldatenko** [comment]
i used https://github.com/coreos/clair image scanner and https://github.com/arminc/clair-scanner for local run.

**Comment by andriisoldatenko** [comment]
Possible fix:

`cat Dockerfile`:

```
FROM docker.elastic.co/elasticsearch/elasticsearch-oss:7.2.0

RUN yum update -y python vim-minimal
```

`docker build -t es-py .` and than rescan again:

```
clair-scanner --ip=192.168.1.199 es-py:latest
2019/07/25 23:36:33 [INFO] ▶ Start clair-scanner
2019/07/25 23:36:45 [INFO] ▶ Server listening on port 9279
2019/07/25 23:36:45 [INFO] ▶ Analyzing 21ce7dd52e6869f92d3a200a18b1dcca939ca431550d8c40f3581e867c0df177
2019/07/25 23:36:45 [INFO] ▶ Analyzing b28806d925ca721d333a35f82d8d36ca4c469b59213ec587f2816f5ab31a5b2c
2019/07/25 23:36:45 [INFO] ▶ Analyzing 89e50d8812763eb50d30b908bb54d7d39a0878447bf974c5f664a29d18f81fae
2019/07/25 23:36:45 [INFO] ▶ Analyzing acb2cea8184b1b8e9e09b92e30c1bf9c1692894d8662ec6b235137a3704f75aa
2019/07/25 23:36:45 [INFO] ▶ Analyzing 9b0b27958d8c1b5327ee34b4e37c7a0835b414328df95fbe7f717bca3242f37d
2019/07/25 23:36:45 [INFO] ▶ Analyzing 963472052b4507f37665bf6119296f228d1505245b02c330a2a36ffcd613b8b7
2019/07/25 23:36:45 [INFO] ▶ Analyzing 32e3639eb42dcf596a95c57f0704e6c286f062327dd417d58d45bbfa23a1213f
2019/07/25 23:36:45 [INFO] ▶ Analyzing 4e7cc8a38cf2ac5b32c6651166a98ffcf013865ab50ecdc899f5e60e2c8e313c
2019/07/25 23:36:50 [INFO] ▶ Image [es-py:latest] contains NO unapproved vulnerabilities
```

**Comment by andriisoldatenko** [comment]
BTW same issue exists in `docker.elastic.co/kibana/kibana-oss:7.2.0`

**Comment by atorok** [comment]
We do have a `yum update -y` in our Dockerfile so it would seem that those are vulnerabilities that popped up after the release of `7.2` that would get addressed when the image is build for `7.2.1`

**Comment by mark-vieira** [comment]
This is likely no longer an issue. We publish using the latest base centos image and any and all security updates inherited from that. As Alpar mentions, these would be vulnerabilities that come up _after_ release and are auto-addressed in the next release.
